src/gui/app.cpp

```diff
@@ -159,6 +163,10 @@ void App::processCommandlineOptions(QCommandLineParser& parser, QStringList argu
 				QCoreApplication::translate("main", "Error if nvim does not responde after count milliseconds"),
 				QCoreApplication::translate("main", "ms"),
 				"20000"));
+	parser.addOption(QCommandLineOption("class",
```

Comment:
I would use `--name` as the argument, just like your kitty example.

What this argument really does is change the application name. `WM_CLASS` is a Linux/X11 specific implementation detail.

Reply:
Yeah there doesn't appear to be a standard. For example, with `qutebrowser` you set it like `--qt-arg name <NAME>`. But `--name` seems reasonable, and matches QT's terminology.

Reply:
Updated argument name and description.